featuretools/primitives/data/free_email_provider_domains.txt

```diff
@@ -0,0 +1,3782 @@
+1033edge.com
```

Comment:
@tuethan1999 this file was taken from here? 
https://github.com/willwhite/freemail/blob/master/data/free.txt